Streetcode/Streetcode.XUnitTest/MediatRTests/Timeline/HistoricalContextTests/GetAllHistoricalContextTest.cs

```diff
@@ -0,0 +1,133 @@
+﻿using Xunit;
+using AutoMapper;
+using Moq;
+using System.Linq.Expressions;
+using Microsoft.EntityFrameworkCore.Query;
+using Streetcode.DAL.Repositories.Interfaces.Base;
+using Streetcode.DAL.Entities.Timeline;
+using Streetcode.BLL.DTO.Timeline;
+using Streetcode.BLL.MediatR.Timeline.HistoricalContext.GetAll;
+using Streetcode.BLL.DTO.Streetcode.TextContent;
+using Streetcode.BLL.MediatR.Streetcode.Fact.GetAll;
+using Streetcode.DAL.Entities.Streetcode.TextContent;
+
+namespace Streetcode.XUnitTest.MediatRTests.Timeline.HistoricalContextTests
+{
+	public class GetAllHistoricalContextTest
+	{
+		public Mock<IRepositoryWrapper> _mockRepository;
+		public Mock<IMapper> _mockMapper;
+
+		public GetAllHistoricalContextTest()
+		{
+			_mockRepository = new();
+			_mockMapper = new();
+		}
+
+		[Fact]
+		public async Task ShouldReturnSuccessfully_CorectType() {
+			//Arrange
+			(_mockMapper, _mockRepository) = GetMapperAndRepo(_mockMapper,_mockRepository);
+			var hendler = new GetAllHistoricalContextHandler(_mockRepository.Object, _mockMapper.Object);
+
+			//Act
+			var result = await hendler.Handle(new GetAllHistoricalContextQuery(), CancellationToken.None);
+
+			//Assert
+			Assert.Multiple(
+				() => Assert.NotNull(result),
+				() => Assert.IsType<List<HistoricalContextDTO>>(result.ValueOrDefault)
+				);
+		}
+
+		[Fact]
+		public async Task ShouldReturnSuccessfully_CountMatch() {
+			//Arrange
+			(_mockMapper, _mockRepository) = GetMapperAndRepo(_mockMapper, _mockRepository);
+			var hendler = new GetAllHistoricalContextHandler(_mockRepository.Object, _mockMapper.Object);
+			
+			//Act
+			var result = await hendler.Handle(new GetAllHistoricalContextQuery(), CancellationToken.None);
+
+			//Assert
+			Assert.Multiple(
+				() => Assert.NotNull(result),
+				() => Assert.Equal(GetListHistoricalContext().Count(), result.Value.Count())
+				);  
+			}
```

Comment:
same here (please be attentive to styling code, this will make it more readable)